cumulusci/core/utils.py

```diff
@@ -96,13 +98,20 @@ def process_glob_list_arg(arg):
 
 def process_list_arg(arg):
     """Parse a string into a list separated by commas with whitespace stripped"""
+    if isinstance(arg, Path):
+        arg = str(arg)
+
     if isinstance(arg, list):
         return arg
     elif isinstance(arg, str):
         args = []
         for part in arg.split(","):
             args.append(part.strip())
         return args
+    else:
+        getLogger(__file__).info(
+            f"Unknown argument type `{type(arg)}` for type `{arg}`"
+        )
```

Comment:
This looks like a confusing message to log. To the user, they are configuring options, not arguments. The option value is referred to as "type" here. And if they got the type wrong, they'd probably like to know which option it was. I like the general idea of warning about unexpected types, but could we take a step back and do some team work on ACs for this before implementing?

Reply:
I agree it can be tightened up and I did do some of that. But it's a code path that is hit so infrequently that it didn't become an issue in the last 4 years. I don't think we should open up a big conversation with a lot of people for it. It's also going to be replaced with the grand argument refactoring at some point.

Reply:
100% chance if we open it up for a larger conversation the logical thing to say is "shouldn't we tell them the arg name" which is not possible in a backwards compatible context. We need to refactor to make this message "good".